admin/resources/js/components/form/Checkbox.tsx

```diff
@@ -42,7 +42,8 @@ const Checkbox: React.FunctionComponent<CheckboxProps> = ({
       hideOptional
     >
       <input
-        style={{ order: -1 }}
+        style={{ order: -1, marginBottom: "0.5rem" }}
```

Comment:
Why do you need to set the `margin-bottom` value the style attribute if you are also doing it in the `data-h2-margin` attribute's value?